Sensus.iOS/Concurrent/MainConcurrent.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright 2014 The Rector & Visitors of the University of Virginia
```

Comment:
Why is there a MainConcurrent.cs in this project as well as in Sensus.iOS.Tools?